README.md

```diff
@@ -1,6 +1,12 @@
 # Beaker tasks used with [skt](https://github.com/RH-FMK/skt) runner
 
-
+## How to run tests
+Here is a list of common prerequisites for all beaker tests, test specific dependencies and steps can be found in the README.md within the test directory.
+~~~
+$ wget -O /etc/yum.repos.d/beaker-client.repo https://beaker-project.org/yum/beaker-client-Fedora.repo
+$ wget -O /etc/yum.repos.d/beaker-harness.repo https://beaker-project.org/yum/beaker-harness-Fedora.repo
+$ dnf install -y beaker-client beakerlib restraint-rhts
```

Comment:
You're using `$` as the shell prompt above. However, that's used by non-privileged users, for which all of the above commands would fail. You could either add `sudo` in front of those commands or switch the prompt to `#`. I'd prefer the former, as people don't normally use the root shell these days.